src/validation.cpp

```diff
@@ -5087,7 +5087,9 @@ bool LoadMempool(CTxMemPool& pool)
         file >> mapDeltas;
 
         for (const auto& i : mapDeltas) {
-            pool.PrioritiseTransaction(i.first, i.second);
+            if(MoneyRange(i.second)){
```

Comment:
```suggestion
            if(MoneyRange(i.second)) {
```

Reply:
Noted..

Reply:
You may install `clang-format` and run the https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy script to preempt whitespace nitpicking.